src/boot/cic6105.c

```diff
@@ -1,3 +1,100 @@
+#include "global.h"
 #include "cic6105.h"
+#include "fault.h"
+#include "regs.h"
+
+// TODO N64 fault.c functions
+void func_800AE1E0_unknown(s32, s32);
+void func_800AE258_unknown(const char*, ...);
+
+s32 func_80001714(void);
+
+struct struct_800067C0 D_800067C0_unknown = {
+    0x00000004,
+    0x00000000,
+    rspbootTextStart,
+    0x000003E8,
+    // this rspbootTextEnd may be func_80006720_unknown ?
+    rspbootTextEnd,
+    0x00000020,
+    gBuildTeam,
+    0x00000008,
+    0x00000000,
+    0x00000000,
+    0x00000000,
+    0x00000000,
+    0x00000000,
+    0x00000000,
+    0x00000000,
+    0x00000000,
+};
 
 u32 B_80008EE0;
+s32 B_80008EE4_unknown;
+FaultClient sCIC6105FaultClient;
```

Comment:
maybe we should call these file `CIC6105.h`/`CIC6105.c` to better match the naming (also MM calls it that, didn't notice until recently)

Reply:
can't say I like uppercase in my filenames

Reply:
I agree with you, just thought I'd throw it out there